var/spack/repos/builtin/packages/mofem-users-modules/package.py

```diff
@@ -43,14 +43,28 @@ class MofemUsersModules(CMakePackage):
 
     homepage = "http://mofem.eng.gla.ac.uk"
     url = "https://bitbucket.org/likask/mofem-joseph/downloads/users_modules_dummy"
-    version('1.0', '5a8b22c9cdcad7bbad92b1590d55edb1', expand=False)
```

Comment:
What happened to this release?

Reply:
In previous version extension, 1.0 worked with all versions of the core library.  However, packages are better to manage if the extension version, i.e. mofem-usres-modules, follows package mofem-cpehas.  

I removed the old version, since ad hoc I put a version of it as 1.0. If it will be left it will be the higher version any other. So spack will install version 1.0 all the times when someone does
spack install mofem-users-modules

Instead of installing the most recent, in this case, mofem-users-modules@0.8.12.